DRAFT_Connectivity.Rmd

```diff
@@ -1,266 +1,280 @@
+
+### Original units {#units-con}
+The infrastructure index has no specific units as it is a measure of landscape fragmentation and disruption. It is expressed as a dimensionless index.
+
+### Temporal coverage {#temporal-coverage-con}
+The data covers three time points: 2003, 2013, and 2023. This provides a 20-year temporal perspective for assessing changes in connectivity over time. The wetland mapping provides information in the form of pixel values representing wetland presence or absence.
+
+### Aditional comments about the dataset {#additional-comments-con}
+It is important to note that infrastructure data are based on available map and geographical data sources. Sometimes, there may be variations and uncertainties in this data that can affect the accuracy of the infrastructure index.
+
+## Ecosystem characteristic {#egenskap-con}
+In the context of the Norwegian standard, the "Connectivity" indicator aligns with the ecosystem characteristic known as **Landskapsøkologiske mønstre**. 
+
+### SEEA EA (UN standard) {#seea-con}
+In the framework of the United Nations System of Environmental-Economic Accounting (SEEA EA), the _Connectivity_ indicator can be classified as a **C1 Landscape and seascape characteristics** indicator.
+
+## Collinearities with other indicators {#collinear-con}
+Connectivity is not thought to exhibit collinearity with any other indicator at the present.
+
+## Reference condition and values {#ref-conditions-and-values-con}
+### Reference condition {#ref-condition-con}
+The reference condition for the "Connectivity" indicator is characterized as one with minimal negative human impact on the connectivity of ecosystems. In this reference condition, the landscape exhibits natural and undisturbed patterns of connectivity between suitable habitats. This state represents the optimal ecological condition for connectivity assessment.
+
+### Reference values, thresholds for defining _good ecological condition_, minimum and/or maximum values {#ref-values-con}
+The reference values for the "Connectivity" indicator are determined by calculating the connectivity or mean distance to mire polygons, when excluding infrastructure from the analysis.
+
+In the report «Vurdering av økologisk tilstand for fjell i Norge i 2021» (Framstad et al. 2021) the value of 0,6 was used as the threshold for good ecological condition. We will tentatively use this threshold value again here. A perpetual discussion about the reference (and threshold values) for the connectivity indicator is maintained on [GitHub.](https://github.com/NINAnor/ecosystemCondition/issues/143)  
```

Comment:
The threshold value was 0.6 in the alpine assesment, and not 0.8. This is in any case quite unsatisfactory, yo use the default value. I create an issue to track this development.